### ISSUE TITLE
Error sending large zips

### ISSUE THREAD

**Issue by jennydaman** [FNNDSC/pfcon#120]
Related to https://github.com/FNNDSC/ChRIS_ultron_backEnd/issues/369

On pfcon v4.0.1 this was supposed to be fixed but I am still getting errors.

First I got this error from our prod instance:

```
[ERROR][manager 10 139702796957504] [CODE04,crispy-139]: Received bad zip file from remote, detail: File is not a zip file
```

I tried replicating this using [dbg-bigfiles](https://github.com/FNNDSC/dbg-bigfiles) locally but got a different error:

```
[ERROR][manager 9 139843096958784] [CODE03,chris-jid-1]: Error fetching zip from pfcon url -->http://pfcon.host:5005/api/v1/<--, detail: ('Connection aborted.', RemoteDisconnected('Remote end closed connection without response'))
```

This happens _sometimes_, I can't reliably reproduce it.

**Comment by jennydaman** [comment]
No changes to services, tried the same thing, got different errors while sending zip TO pfcon:

```
[ERROR][manager 10 139702796957504] [CODE01,crispy-144]: Error submitting job to pfcon url -->http://galena.tch.harvard.edu:5005/api/v1/<--, detail: ('Connection aborted.', RemoteDisconnected('Remote end closed connection without response'))
[INFO][manager 10 139702796957504] Saving job crispy-144 DB status as 'cancelled'
[INFO][manager 10 139702796957504] Saving job crispy-144 DB end_date as '2022-03-09 18:15:56.907335+00:00'
[INFO][trace 10 139702796957504] Task plugininstances.tasks.run_plugin_instance[4207267f-9e0a-4c53-a60a-66fcf6acc25f] succeeded in 777.6472116503865s: None
[ERROR][manager 12 139702796957504] [CODE01,crispy-147]: Error submitting job to pfcon url -->http://galena.tch.harvard.edu:5005/api/v1/<--, detail: ('Connection aborted.', RemoteDisconnected('Remote end closed connection without response'))
[INFO][manager 12 139702796957504] Saving job crispy-147 DB status as 'cancelled'
[INFO][manager 12 139702796957504] Saving job crispy-147 DB end_date as '2022-03-09 18:15:57.814875+00:00'
[INFO][trace 12 139702796957504] Task plugininstances.tasks.run_plugin_instance[e9017001-049c-4ac5-9199-ceb79d0e4d5d] succeeded in 774.5826649032533s: None
[ERROR][manager 11 139702796957504] [CODE01,crispy-145]: Error submitting job to pfcon url -->http://galena.tch.harvard.edu:5005/api/v1/<--, detail: ('Connection aborted.', RemoteDisconnected('Remote end closed connection without response'))
[INFO][manager 11 139702796957504] Saving job crispy-145 DB status as 'cancelled'
[INFO][manager 11 139702796957504] Saving job crispy-145 DB end_date as '2022-03-09 18:15:59.932765+00:00'
[INFO][trace 11 139702796957504] Task plugininstances.tasks.run_plugin_instance[5b00c797-8f92-412a-a510-9585d10c2bf9] succeeded in 778.8244837056845s: None
[ERROR][manager 9 139702796957504] [CODE01,crispy-146]: Error submitting job to pfcon url -->http://galena.tch.harvard.edu:5005/api/v1/<--,detail: ('Connection aborted.', RemoteDisconnected('Remote end closed connection without response'))
[INFO][manager 9 139702796957504] Saving job crispy-146 DB status as 'cancelled'
[INFO][manager 9 139702796957504] Saving job crispy-146 DB end_date as '2022-03-09 18:16:00.041027+00:00'
[INFO][trace 9 139702796957504] Task plugininstances.tasks.run_plugin_instance[da40c0f1-00f2-4841-a4d2-f1dabae5845f] succeeded in 777.8498466014862s: None
```

Meanwhile, pfcon logs:

```
[2022-03-09 18:13:33,769] [INFO][resources:72 12 140438515623744] Received job crispy-144
[2022-03-09 18:13:34,793] [INFO][mount_dir:30 12 140438515623744] 2498 files to decompress for job crispy-144
[2022-03-09 18:13:35,514] [INFO][resources:72 10 140438515623744] Received job crispy-147
[2022-03-09 18:13:35,551] [INFO][mount_dir:30 10 140438515623744] 2498 files to decompress for job crispy-147
[2022-03-09 18:13:36,631] [INFO][resources:72 9 140438515623744] Received job crispy-146
[2022-03-09 18:13:36,675] [INFO][mount_dir:30 9 140438515623744] 2498 files to decompress for job crispy-146
[2022-03-09 18:13:37,070] [INFO][resources:72 11 140438515623744] Received job crispy-145
[2022-03-09 18:13:37,105] [INFO][mount_dir:30 11 140438515623744] 2498 files to decompress for job crispy-145
```

**Comment by jennydaman** [comment]
Fixed by ba6444c4c2a0a716c639e8495a70035e8ac91c7a
